knowledge-base/piechart-label-customization.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Customizing labels in the Pie Chart
+page_title: Customizing Pie Charts labels - .NET MAUI Knowledge Base
+description: Learn how to customize the Telerik UI for .NET MAUI Pie Chart labels.
+type: how-to
+slug: piechart-label-customization
```

Comment:
You may want to add a link to this KB in some of the related Chart docs. This sounds like a common scenario and linking it should be helpful for the clients.